api/data_refinery_api/views/stats.py

```diff
@@ -32,7 +32,7 @@
 )
 from data_refinery_common.utils import get_active_volumes, get_nomad_jobs_breakdown
 
-JOB_CREATED_AT_CUTOFF = datetime(2019, 6, 5, tzinfo=timezone.utc)
+JOB_CREATED_AT_CUTOFF = datetime(2019, 9, 19, tzinfo=timezone.utc)
```

Comment:
Any particular reason for this date?

Reply:
This matches the date on the backend, I think that's why the dashboard shows so many pending jobs that don't exist